recipes/opentelemac/meta.yaml

```diff
@@ -1,30 +1,110 @@
-{% set name = "opentelemac" %}
-{% set version = "v8p4r0" %}
-{% set build = 0 %}
-{% set mpi = "openmpi" %}
-
-package:
-  name: {{ name|lower }}
-  version: {{ version }}
-
-source:
-  - url: https://gitlab.pam-retd.fr/otm/telemac-mascaret/-/archive/{{ version }}/telemac-mascaret-{{ version }}.tar.gz
-    folder: opentelemac
-    sha256: cfa2c05e41b28ba8aaea19b4b483e6d848bb08f823c8526c0824953a1cb4d564
-
-about:
-  home: http://www.opentelemac.org/
-  license: GPLv3
-  license_file: opentelemac/LICENSE.txt
-  summary: openTelemac
-  description: |
-    The TELEMAC-MASCARET system is an integrated suite of solvers for use in the
-    field of free-surface flow.
-  doc_url: http://wiki.opentelemac.org/doku.php?id=documentation_{{ version }}
-  dev_url: https://gitlab.pam-retd.fr/otm/telemac-mascaret/
-
-extra:
-  recipe-maintainers:
-    - nicogodet
-    - brey
-    - tomsail
\ No newline at end of file
+{% set name = "opentelemac" %}
+{% set version = "v8p4r1" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # - url: https://gitlab.pam-retd.fr/otm/telemac-mascaret/-/archive/{{ version }}/telemac-mascaret-{{ version }}.tar.gz
+  - url: https://github.com/nicogodet/otm-sources/releases/download/{{ version }}/telemac-mascaret-{{ version }}.tar.gz
```

Comment:
why are you mirroring from your repo?

Reply:
For a stripped source tarball excluding examples, documentation and notebooks (4mb against >1Gb tarball)

Reply:
and http://www.opentelemac.org/index.php/kunena/feedback-and-suggestions/14015-provide-a-smaller-source-code-archive-on-gitlab